app/assets/javascripts/application.js

```diff
@@ -234,6 +240,7 @@ function template_info(div, url) {
     },
     error: function(jqXHR, textStatus, errorThrown) {
       $(div).html('<div class="alert alert-warning alert-dismissable">' +
+        '<span class="pficon-layered"><span class="pficon pficon-warning-triangle"></span><span class="pficon pficon-warning-exclamation"></span></span>' +
```

Comment:
does it make sense to have an icon helper? similar to https://github.com/theforeman/foreman/pull/2971/files#diff-a9c3bd311eab80c9ebe6a69830f9ad02R560